manager2/src/app/admin/project/project.component.ts

```diff
@@ -83,13 +86,20 @@ export class ProjectComponent implements OnInit {
             resp => this.all_users = resp,
             err => console.log('failed to get all users')
         );
+        this.configService.config.subscribe(
+            resp => {
+                this.config = resp;
+            },
+            err => console.log('failed to get config')
+        );
+
     }
 
     show_project_users(projectId) {
         this.projectsService.get(projectId).subscribe(
             resp => {
-                resp.expire = this.date_convert(resp.expire);
                 this.project = resp;
+                this.project.expire = (resp.expire) ? this.date_convert(resp.expire) : this.date_convert(new Date());
```

Comment:
Maybe we should force expiration (and validate format before sending project creation/update

A project should always have an expiration, even if very far.
Here, if there is no expiration, it will be set to current date, can be misleading

Reply:
you are ritgh, I will add default expire value